apps/menu-bar/src/utils/device.ts

```diff
@@ -18,11 +18,14 @@ export type BaseDevice = {
 );
 
 export function getDeviceOS(device: Device): 'android' | 'ios' {
+  if (device.osType === 'tvOS') {
+    return 'ios';
+  }
```

Comment:
why not returning tvOS here?

Reply:
Adding a new `platform` would be a large change that would touch many files, and did not seem to be needed for an experimental feature.

Reply:
Got it, make sense to me